CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   overridden by the `MicVolumeIndicator` component `className`.
 - NotificatonGroups don't accept pointer-events.
 - Clean up timeouts when `useFocusIn` and `useMouseMove` hooks are unmounted.
+- Fix removeEventListener bug for WithTooltip
```

Comment:
Since, we released and no one has commited we do not have a new [Unreleased] section, check [this](https://github.com/aws/amazon-chime-sdk-component-library-react/pull/438/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed) PR and add the [Unreleased] section with your fix under the "Fixed" section.

Nit: Recently, we have started adding a "." at the end of sentence in the CHANGELOG in sync with the JS SDK.